tests/unittest_manager.py

```diff
@@ -315,5 +316,13 @@ def test_borg(self) -> None:
         self.assertIs(built, second_built)
 
 
+class ClearCacheTest(unittest.TestCase, resources.AstroidCacheSetupMixin):
```

Comment:
Can we make this `pytest`-based? If it is too much of a hassle never mind!

Reply:
Yeah, I guess I'd kind of prefer not to.